lib/eslint/eslint.js

```diff
@@ -364,7 +364,7 @@ async function loadFlatConfigFile(filePath, hasUnstableTSConfigFlag) {
             throw new Error("The 'jiti' library is required for loading TypeScript configuration files. Make sure to install it.");
         });
 
-        const jiti = createJiti(__filename);
+        const jiti = createJiti(__filename, { moduleCache: false });
```

Comment:
Can you add a comment here describing what we're doing? Specifically, what does `moduleCache: false` do?

Reply:
Disabling `moduleCache` allows importing the same config multiple times on the same process (kinda HMR effect) with the cost of fs (cache) read and eval (only ts files, the native imports in the tree including `.mjs` imports, remain cached)